src/main/java/com/teamc/fms/mapper/TenantMapper.java

```diff
@@ -0,0 +1,19 @@
+package com.teamc.fms.mapper;
+
+import com.teamc.fms.dto.TenantDto;
+import com.teamc.fms.entity.Tenant;
+import org.modelmapper.ModelMapper;
+
+public class TenantMapper {
+
+    private ModelMapper mapper;
```

Comment:
mapper will be null, as no injection happened
please make it private final
and add @RequiredArgsConstructor